code/modules/surgery/surgery_step.dm

```diff
@@ -76,7 +77,8 @@
 		implement_speed_mod = implements[implement_type] / 100.0
 
 	speed_mod /= (get_location_modifier(target) * (1 + surgery.speed_modifier) * implement_speed_mod)

```

Comment:
rename the `speed_mod` var to `bad_speed_mods` (do this here and in its definition)
put a comment on it saying "numbers 0<x<1 here = slower surgery"

Reply:
sure

Reply:
decided not to do this since i'm not doing the below and don't want people to think there's a good one somewhere in the code ether.